conda/recipes/dask-cuda/meta.yaml

```diff
@@ -32,7 +32,7 @@ requirements:
     - pynvml>=11.0.0
     - numpy>=1.16.0
     - numba>=0.53.1
-    - click<8.1
+    - click==8.0.4
```

Comment:
We discussed offline and decided to match what was done in PR ( https://github.com/rapidsai/integration/pull/453 ). The `distributed` issue this stems from is a bit different than the one that `black` encountered, but this pinning should work just as well here